src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#image">
+        <img
+        class="image"
+        src="images\moyo_1.png"
+        alt="logo"
+        >
```

Comment:
```suggestion
        <img
          class="image"
          src="images\moyo_1.png"
          alt="logo"
        >
```

If you have three or more attributes then each of them should be in a new line with two more spaces the opening tag and closing bracket should be with the same amount of spaces as opening tag